src/components/About.js

```diff
@@ -1,18 +1,38 @@
-import React from "react";
+import React, { useRef, useState, useEffect } from "react";
 import { useHistory } from "react-router-dom";
 
 import textStyles from "./TextSection/text-section.module.css";
 import imgStyles from "../css-modules/photo-text-section.module.css";
 import newsletterStyles from "../components/Newsletter/newsletter.module.css";
 
+const cx = require("classnames");
+
 export default function About(props) {
+  const placeholder = useRef();
+  const [showImage, setShowImage] = useState(false);
+  const placeholderStyle = cx("placeholder", { hidePlaceholder: showImage });
+
   let history = useHistory();
-  console.log(history);
 
   function handleBtnRedirect() {
     history.push("/about");
   }
 
+  useEffect(() => {
+    const callback = (entries) => {
+      entries.forEach((entry) => {
+        if (entry.isIntersecting) {
+          setShowImage(true);
```

Comment:
It seems you understood the basic concept of IO, well done!

So, with this approach, the image is rendered and loaded, but we still see a "blank space" while waiting for the image to load (noticeable in slow connections)

We can go a step further with IO. The idea would be to use IO to load the image behind the scenes and only after `setImgVisible` to true when the image is loaded.

Take a look [at this tutorial](https://blog.teamtreehouse.com/learn-asynchronous-image-loading-javascript), and adapt it to React.


<details>
<summary>see solution (after you give it  atry!)</summary>

```js
import imgSrc from "../img/about.jpg";

// ...
      entries.forEach((entry) => {
        if (entry.isIntersecting) {
          var downloadingImage = new Image();
          downloadingImage.onload = function () {
            setShowImage(true);
          };
          downloadingImage.src = imgSrc;
        }
      });

// ...

showImage ?  <img className={imgStyles.img} src={imgSrc} /> : xxx

// ...


``´
</details>